cfgov/ask_cfpb/documents.py

```diff
@@ -20,7 +20,7 @@ class AnswerPageDocument(Document):
 
     def get_queryset(self, *args, **kwargs):
         query_set = super().get_queryset(*args, **kwargs)
-        return query_set.filter(live=True, redirect_to_page=None)
+        return query_set.filter(live=True)
```

Comment:
I think you could simplify this method to just `return super().get_queryset(*args, **kwargs).live()`